Emby.Server.Implementations/Playlists/PlaylistManager.cs

```diff
@@ -196,26 +196,37 @@ private void AddToPlaylistInternal(string playlistId, IEnumerable<Guid> itemIds,
             // Retrieve all the items to be added to the playlist
             var items = GetPlaylistItems(itemIds, playlist.MediaType, user, options)
                 .Where(i => i.SupportsAddingToPlaylist)
-                .ToList();
+                .ToArray();
```

Comment:
Why?

Reply:
Changed it to match the data type for `uniqueItems` below, but also since the size of this collection doesn't change, creating an array has less overhead than creating a list.

Reply:
> creating an array has less overhead than creating a list.
The opposite is true, since the length isn't known in advance the collection will grow until it contains all elements, this is fine for `ToList`, but `ToArray` may need to make another allocation + copy to be the exact size.

Reply:
Good catch...updated!